.travis.yml

```diff
@@ -102,7 +102,7 @@ install:
     # Install numpy, flake
     - conda create -n testenv --yes --quiet pip python=$PYTHON > ${REDIRECT_TO};
     - source activate testenv > ${REDIRECT_TO};
-    - conda install --yes --quiet numpy$NUMPY nose pytest > ${REDIRECT_TO};
+    - conda install --yes --quiet numpy$NUMPY nose pytest networkx > ${REDIRECT_TO};
```

Comment:
You will probably need to add both `networkx` and `numpy$NUMPY` to the `conda install` line, too, to make sure `networkx` doesn't pull in the wrong version via recommendation